### DISCUSSION TITLE
Calibration (Camera intrinsic and camera-lidar extrinsic) related questions

### DISCUSSION THREAD

**Discussion by allenthreee** [larics/icuas24_competition#53] [Q&A]
I guess the QR code pattern in the calibration bag is downloaded from [Kalibr repo](https://github.com/ethz-asl/kalibr/wiki/downloads) via [this link](https://drive.google.com/file/d/1DqKWgePodCpAKJCd_Bz-hfiEQOSnn_k0/view) and print it by A4 paper?
Am I right? What about the exact size of these grid cells?
![frame000073](https://github.com/larics/icuas24_competition/assets/59171742/9c72bdc4-dc6f-40e0-94ab-5f2c1e525b23)
![kalibr_download](https://github.com/larics/icuas24_competition/assets/59171742/32eb43f1-4d00-4f83-bed1-ac3a9ce315a5)

**Comment by fpetric** [comment]
Will check in the lab for the exact size. It is the kalibr file on A4.
